### DISCUSSION TITLE
Undefined control sequence. \pyif

### DISCUSSION THREAD

**Discussion by jmadkour** [tndrle/PyLuaTeX#12] [Q&A] [answered]
Hello everybody
I'm trying to check the \pyif statement but I get the following error message:
Undefined control sequence. \pyif
Can someone help.
Thanks.
[file.txt](https://github.com/tndrle/PyLuaTeX/files/8923253/file.txt)
[test.log](https://github.com/tndrle/PyLuaTeX/files/8923254/test.log)

**Comment by tndrle** [comment, marked as answer]
Your log file says

> Package: pyluatex 2021/11/05 v0.4.0

The `\pyif` command is available in version 0.5.0, so you have to update PyLuaTeX.

**Reply by jmadkour** [reply]
Thank you so much. It works!
